addons/core/tests/unit/services/scope-test.js

```diff
@@ -10,9 +10,21 @@ module('Unit | Service | scope', function (hooks) {
     assert.ok(service.org, 'Service contains org scope');
   });
 
+  test('it can contain orgs list scope', function (assert) {
+    let service = this.owner.lookup('service:scope');
+    service.orgsList = {};
```

Comment:
I sort of expect orgs list to be an array type.  Why is it an object in the tests?

Reply:
@randallmorey 
`projects` and `orgs` are an object because when querying them, within `scopes/scope` route it returns the model class which is an object.

Even accessing the class such as `orgs.content` returns an `object` type.

Happy to discuss this over slack or zoom if you like.

Reply:
@randallmorey Changed